src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1020,14 +1027,19 @@ namespace NServiceBus.ConsistencyGuarantees
 }
 namespace NServiceBus.Container
 {
+    [System.ObsoleteAttribute("The NServiceBus dependency injection container API has been deprecated. Use the e" +
+        "xternally managed container mode to use custom containers. Will be removed in ve" +
+        "rsion 9.0.0.", true)]
     public class ContainerCustomizations
     {
-        public NServiceBus.Settings.SettingsHolder Settings { get; }
```

Comment:
Wouldn't this member have to stay or doesn't it matter because it is obsoleted with error anyway?

Reply:
I think the property isn't needed directly by the user as customizations would provide their own extension methods. It probably doesn't matter anyway as you can't use `UseContainer` anymore anyway and there will be no container package that targets ContainerCustomizations available for v8.